test/integration/volume/persistent_volumes_test.go

```diff
@@ -1058,6 +1058,25 @@ func TestPersistentVolumeProvisionMultiPVCs(t *testing.T) {
 	defer testClient.CoreV1().PersistentVolumes().DeleteCollection(context.TODO(), metav1.DeleteOptions{}, metav1.ListOptions{})
 	defer testClient.StorageV1().StorageClasses().DeleteCollection(context.TODO(), metav1.DeleteOptions{}, metav1.ListOptions{})
 
+	// Watch all events in the namespace, and save them to artifacts for debugging.
+	// TODO: This is a temporary solution to debug flaky tests `panic: test timed out after 10m0s`.
+	// We should remove this once https://github.com/kubernetes/kubernetes/issues/124136 is fixed.
+	go func() {
+		w, err := testClient.EventsV1().Events(ns.Name).Watch(tCtx, metav1.ListOptions{})
```

Comment:
you probably want to add AllowWatchBookmarks here to help keep the watch from closing